server/views/components/actions-card/template.njk

```diff
@@ -1,8 +1,37 @@
-<div class="actions-list">
-  <ul class="govuk-list actions-list-list">
-    <span class="govuk-heading-m actions-list-header">Actions</span>
-    {% for action in actions %}
-      <li><a class="govuk-link" {% if action.dataQa %} data-qa="{{ action.dataQa }}" {% endif %}href="{{ action.href }}">{{ action.title }}</a></li>
-    {% endfor %}
-  </ul>
-</div>
+{% from "govuk/macros/attributes.njk" import govukAttributes %}
+
+{% if params.actions | length %}
+
+  {#- Set classes for this component #}
+  {%- set classNames = "govuk-summary-card actions-card" -%}
```

Comment:
Line 6 to line 12 is exactly how GDS components do this, and is pretty much a copy and paste from one of them (GDS Button I think, but it was a copy from one of them)